internal/pkg/cli/pipeline_update.go

```diff
@@ -40,10 +41,9 @@ const (
 	fmtPipelineUpdateExistPrompt = "Are you sure you want to update an existing pipeline: %s?"
 )
 
-const (
-	connectionsURL = "https://console.aws.amazon.com/codesuite/settings/connections"
-	fmtPipelineURL = "https://%s.console.aws.amazon.com/codesuite/codepipeline/pipelines/%s/view?region=%s"
-)
+const connectionsURL = "https://console.aws.amazon.com/codesuite/settings/connections"
+
+var bbRepoExp = regexp.MustCompile(`(https:\/\/(.+)@bitbucket.org\/)(?P<owner>.+)\/(?P<repo>.+)`)
```

Comment:
Can we add an example for this? Even just a comment with a sample bb url would help readers understand what the first capturing group (`(.+)@bitbucket.org`) is doing

Reply:
Yup, done.